pages/guides/_search.html

```diff
@@ -0,0 +1,23 @@
+<div class="input-group">
+    <div class="input-group-btn" >
+        <div class="btn-group" role="group"> 
+            <button type="button" class="btn btn-primary" >
+                <span class="glyphicon glyphicon-search" aria-hidden="true"></span>
+            </button>
+        </div>
+    </div><input id="search-box" type="text" class="form-control" placeholder="Search guides....">
+    <script>
+        window.guideEntries = [
+            {% for post in site.posts %} 
+            {
+                "title": "{{ post.title | xml_escape }}",
+                "image": "{{ post.image }}",
```

Comment:
Seems like you should call `strip_html`, `strip_newlines`, `jsonify` on each of these?

Reply:
Yes, done that [here](https://github.com/gruntwork-io/gruntwork-io.github.io/pull/147/files#diff-a77f40945e9bcc4da6be5f4e258b3421R4)